samples/tab-deeplink-all-features/nodejs/README.md

```diff
@@ -0,0 +1,119 @@
+---
+page_type: sample
+description: Microsoft Teams sample app for demonstrating deeplink all features using Tab
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "26/01/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-deeplink-all-features-nodejs
+
+---
+ # DeepLink All Features
+
+ This sample app for demonstrating for deeplink all features using Tab.
```

Comment:
This sample app for demonstrating for deep links features using tab.

Reply:
fixed